src/views/Login.scss

```diff
@@ -8,17 +8,17 @@
     height: 100vh;
 }
 
+.login-image {
+    background-color: #fff;
+    background-image: url('../images/LoginImage.svg');
```

Comment:
Point of interest - use SVG instead so it will scale better on different sized displays

Reply:
This image is not centered, it is cropped a little bit on the left, at least on my display